daphne/src/messages.rs

```diff
@@ -4,22 +4,30 @@
 //! Messages in the DAP protocol.
 
 use prio::codec::{
-    decode_u16_items, decode_u32_items, encode_u16_items, encode_u32_items, CodecError, Decode,
-    Encode,
+    decode_u16_items, decode_u24_items, decode_u32_items, decode_u8_items, encode_u16_items,
+    encode_u24_items, encode_u32_items, encode_u8_items, CodecError, Decode, Encode,
 };
 use serde::{Deserialize, Serialize};
 use std::{
     convert::{TryFrom, TryInto},
     fmt::Debug,
     io::{Cursor, Read},
+    str::from_utf8,
 };
+use url::Url;
 
 const KEM_ID_X25519_HKDF_SHA256: u16 = 0x0020;
 const KEM_ID_P256_HKDF_SHA256: u16 = 0x0010;
 const KDF_ID_HKDF_SHA256: u16 = 0x0001;
 const AEAD_ID_AES128GCM: u16 = 0x0001;
 const QUERY_TYPE_TIME_INTERVAL: u8 = 0x01;
 const QUERY_TYPE_FIXED_SIZE: u8 = 0x02;
+const PRIO3_AES128_COUNT: u32 = 0x00000000;
```

Comment:
Hrm, I think we want to try avoid jamming all of these constants into the same list. This PR might be a good time to get in the habit of grouping constants by type.
```suggestion

// VDAFs (draft-wang-ppm-dap-taskprov-00)
const PRIO3_AES128_COUNT: u32 = 0x00000000;
```

Reply:
I grouped these together and also added a VDAF_TYPE_ prefix